google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
Ahh, right you are, `this.children[0]` will not pick up content that is just text in shadow dom.

Well I’m fairly certain the idea behind adding the `div` was to be able to get a single element that could be moved into the infowindow content in one step, minimizing content change events. The way the infowindow content is currently populated with  `this.innerHTML.trim()` leaves event bindings behind, of course.  

The ink issue applies only to this pull request, not to the released code base, so I don’t think a new issue is appropriate.

While it may be possible to loop through `this.childNodes` to set the infowindow content and control for the content change events during the assembly, I’m now doubtful that issue [#288](https://github.com/GoogleWebComponents/google-map/issues/288) can be resolved with infowindow markup as content of the marker.  Rapid mutation, such as the ink example, is just one reason.. There are alternatives to using the infowindow to display information on top of a map.